modules/persistent-storage-csi-vsphere-increase-max-vols-per-node-overview.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-vsphere.adoc
+//
+
+:_mod-docs-content-type: CONCEPT
+[id="persistent-storage-csi-vsphere-increase-max-vols-per-node-overview_{context}"]
+= Increasing maximum volumes per node for vSphere
+
+For VMware vSphere version 7, {product-title} restricts the maximum number of volumes per node to 59. 
+
+However, for vSphere version 8 or later, you can increase the allowable number of volumes per node to a maximum of 255. Otherwise, the default value remains at 59.
+
+.Limitations
+* You must be running VMware vSphere version 8 or later.
+
+* You must have an homogeneous vSphere 8 environment that only contains ESXi 8 hypervisors. Heterogeneous environments that contain a mix of ESXi 7 and 8 are not allowed.
```

Comment:
`mix of ESXi 7 and 8 are not allowed` -> 7 and 8 is not the only combination that is not allowed, if any of the nodes is on version less than 8 - it is not allowed.

And thinking if we need a warning block around here, since we will degrade the cluster if this requirement is not met and users set values above 59: https://github.com/openshift/enhancements/pull/1748/files#diff-a45d3a5e19d424ae3457a5db5722c6b7511bf3662941972a2d8db77edf272f7dR114-R117

Reply:
agree just in case people are still using v6...
+1 add a orange warning stating that failling to comply will degrade the cluster